src/pages/patientView/copyNumberAlterations/CopyNumberTableWrapper.tsx

```diff
@@ -287,9 +289,62 @@ export default class CopyNumberTableWrapper extends React.Component<
                     this.pageStore.cnaCivicVariants
                 );
             },
+            visible:
```

Comment:
this could be problematic because it will cause layout to jump.   can you test whether that's case?  I guess it already re-sorts the data when annotations load, right?!